src/components/form/form.js

```diff
@@ -0,0 +1,50 @@
+import Component from '../../app/js/base/Component';
+
+class Form extends Component {
+    constructor(element) {
+        super(element);
+
+        // Your code here
+        const button = document.querySelector('.form__content__button');
```

Comment:
⛔️ `document.querySelector('.form__content__button')`

✅ `this.getElement('content__button')`